File: backend/src/models/data/InitFlightData.ts
import {Ident} from "./Ident";
import {Telemetrie} from "./Telemetrie";
import {AircraftState} from "./AircraftState";

class InitFlightData{
    public Ident: Ident = new Ident;
    public Telemetrie: Telemetrie = new Telemetrie;
    public State: AircraftState = new AircraftState;
}

export { InitFlightData };

File: client/Bombathlon/Bombatlon/Model/InitFlightData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bombatlon.Model
{
    public class InitFlightData
    {
        public Ident Ident { get; set; }
        public Telemetrie Telemetrie { get; set; }
        public AircraftState State { get; set; }
    }
}
